src/routes.js

```diff
@@ -5,13 +5,35 @@ const index = require('./routes/index');
 const authors = require('./routes/authors');
 const books = require('./routes/books');
 const booksForAuthors = require('./routes/booksForAuthors');
+const session = require('./routes/session');
 
 const router = new KoaRouter();
 
+router.use(async(ctx,next)=>{
```

Comment:
Aquí dejo una sugerencia de estilos (que nuevamente es algo que identifica el linter, pero siempre es bueno tenerlo en mente):
```suggestion
router.use(async (ctx, next) => {
```